packages/amplify-provider-awscloudformation/src/aws-utils/aws-s3.ts

```diff
@@ -70,11 +70,12 @@ export class S3 {
    * Populate the uploadState member with the Amplify deployment bucket name
    */
   private populateUploadState(): void {
-    const projectDetails = this.context.amplify.getProjectDetails();
+    const { amplifyMeta } = this.context.amplify.getProjectDetails();
+    const teamProviderInfo = stateManager.getTeamProviderInfo();
```

Comment:
Should this be updated to use the new parameter manager?

Reply:
this part can't be migrated yet because it's reading from the providers block of the TPI file which is not part of the param manager